src/api/validator.ts

```diff
@@ -43,5 +44,6 @@ export const getValidators = async (
     console.error("validators - backend error:", response.statusText);
   }
 
-  return response.json();
+  const data = await response.json();
+  return data as ValidatorResponse; // Ensure the response is correctly typed
```

Comment:
Nice try :smile: it's a good thing to have things typed :+1: unfortunately this doesn't change much. The type is already provided in the function's definition (we also call it the function's _signature_), check L19:
```
export const getValidators = async (
  chainId: string,
  offset?: number,
  limit?: number
): Promise<ValidatorResponse> => {
```

`Promise<ValidatorResponse>` is the return type of the function, so it's already set.